src/librustc/diagnostics.rs

```diff
@@ -168,6 +168,11 @@ match x {
 ```
 "##,
 
+E0010: r##"
+The value of static and const variables must be known at compile time, and they
```

Comment:
Or maybe?

`Both constants and statics are given values at compile-time. In the case of constants, the value is fixed for the duration of the program's execution, while statics are given an initial value in the program binary.`